### DISCUSSION TITLE
500 SERVER ERROR when trying to import from Nordigen

### DISCUSSION THREAD

**Discussion by LM3469** [firefly-iii/firefly-iii#5524] [Firefly III] [answered]
When trying to import from Nordigen using FIDI, I get a server error 500 right after bank authentification with this error in the log:
`[2022-01-11 13:57:25] production.ERROR: App\Services\Nordigen\Model\Account::setOwnerAddressUnstructured(): Argument #1 ($ownerAddressUnstructured) must be of type array, string given, called in /var/www/html/app/Services/Nordigen/Services/AccountInformationCollector.php on line 117 {"exception":"[object] (TypeError(code: 0): App\\Services\\Nordigen\\Model\\Account::setOwnerAddressUnstructured(): Argument #1 ($ownerAddressUnstructured) must be of type array, string given, called in /var/www/html/app/Services/Nordigen/Services/AccountInformationCollector.php on line 117 at /var/www/html/app/Services/Nordigen/Model/Account.php:359)`

Does anyone have a clue on how to solve this issue?

I have discovered Firefly III last week and it looks perfect for what I want to do with date import from my bank accounts !
Any help will be much appreciated.
(I am also new on Github so please tell me if I should give you any extra information)

**Comment by JC5** [comment, marked as answer]
It's a bug. Tonight, I hope to release a new version that fixes it!

**Reply by LM3469** [reply]
Hi @JC5 I see that you released a the 0.6.3 version of FIDI but I still have a server error 500 right after bank authentification with this error in the log:
`production.ERROR: Undefined array key "ownerAddressUnstructured" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"ownerAddressUnstructured\" at /var/www/html/app/Services/Nordigen/Services/AccountInformationCollector.php:117)`

Is it still a bug?
Thanks

**Reply by JC5** [reply]
https://github.com/firefly-iii/firefly-iii/issues/5526
